### DISCUSSION TITLE
Mac OS Sequoia 15.0 no finder extension?

### DISCUSSION THREAD

**Discussion by ghost** [iterate-ch/cyberduck#16251] [Q&A]
Is there no way to activate the Finder extension under Mac OS 15.0 Sequoia?
I can't find an entry for Mountain Duck.

**Comment by dkocher** [comment]
FinderSync extensions can no longer be found in Settings of macOS apparently [^1]. We have filed _FB14925887_ with Apple. Nevertheless the Mountain Duck Finder Extension should be loaded by the system and work as expected in Finder providing custom icon badges and the context menu.

[^1]: https://developer.apple.com/forums/thread/756711

**Comment by dkocher** [comment]
Also discussed in [^1].

[^1]: aonez/Keka#1464
